source/services/odnoklassniki.js

```diff
@@ -7,9 +7,7 @@ var config = require('../config'),
     dom    = require('../dom');
 
 var odnoklassniki = {
-    counterUrl: config.secure 
-        ? undefined 
-        : 'http://connect.ok.ru/dk?st.cmd=extLike&ref={url}&uid={index}',
+    counterUrl: config.protocol + '://connect.ok.ru/dk?st.cmd=extLike&ref={url}&uid={index}',
```

Comment:
@vitkarpov Good point! Unfortunately, currently we use `config.protocol` in all services, so omitting it here in favor of `//` would be inconsistent. However, if you’d like to get rid of `config.protocol` in all services, feel free to submit a PR :-)